pages/index.js

```diff
@@ -410,27 +410,25 @@ export default function Home({ total, today }) {
                 {features.map((feature) => (
                   <div key={feature.name} className="pt-6">
                     <div className="flow-root rounded-lg bg-gray-50 px-6 pb-8">
-                      <div className="-mt-6">
-                        <div>
-                          <span className="inline-flex items-center justify-center rounded-xl bg-indigo-500 p-3 shadow-lg">
-                            <feature.icon
-                              className="h-8 w-8 text-white"
-                              aria-hidden="true"
-                            />
-                          </span>
-                        </div>
-                        <h3 className="mt-8 text-lg font-semibold leading-8 tracking-tight">
-                          <Link
-                            href={feature.path}
-                            className="text-gray-900 hover:text-indigo-600 hover:underline"
-                          >
+                      <Link aria-label="Go to ${feature.name} page"
```

Comment:
![image](https://user-images.githubusercontent.com/95094057/213867350-6039a3f5-92c4-49cb-8232-19f09024e1cf.png)

Make it like this 
```
aria-label={`Go to ${feature.name} page`}
```